xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/job/question/XClassBreakingQuestion.js

```diff
@@ -0,0 +1,140 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+/*!
+#set ($jsExtension = '.min.js')
+#if (!$services.debug.minify)
+  #set ($jsExtension = '.js')
+#end
+*/
```

Comment:
Not sure if this is supposed to be in a JS comment.

Reply:
I get this code from ExtensionBreakingQuestion: I think the comments are on purpose to avoid lint checking errors. But I might be wrong.

Reply:
> I think the comments are on purpose to avoid lint checking errors

Yep. Took me several tries to make it right for ExtensionBreakingQuestion :)

Reply:
It's actually to prevent the comment from being removed when the JavaScript is minified.